public/jira/derived/work-timing/work-timing.ts

```diff
@@ -29,8 +29,8 @@ export type DerivedWorkTiming = {
   startAndDueDateDaysOfWork: number | null;
   hasSprintStartAndEndDate: boolean;
   sprintDaysOfWork: number | null;
-  sprintStartData: StartData;
-  endSprintData: DueData;
+  sprintStartData: StartData | null;
```

Comment:
Everything else here is `null` so this is a good change. And this may be due to CanJS, in which case it should stay this way. However...

I generally avoid using `null` as an optional value, only using it to mean "this value intentionally left blank". Many APIs use null so you know it's empty and not a bug (particularly GraphQL), which I appreciate, though I turn those to undefined in my fetch post-processing.

Why, in no particular order:
* Easier to skim `foo?: string`
* Follows the existing patterns established by official JS APIs
* More type-safe when working with external data. If data is missing (on accident), it'll be`undefined`.